hack/generate.sh

```
@@ -9,13 +9,35 @@ PACKAGE=github.com/kubevirt/hyperconverged-cluster-operator
+  local tool=$1
+  local tool_name
+  tool_name="$(echo "${tool}" | sed -E 's|^.+/([^/@]+)@.+$|\1|')"
 
-go install \
-	"k8s.io/kube-openapi/cmd/openapi-gen@${KUBEOPENAPI_VER}"
+  local req_version
+  req_version="$(echo "${tool}" | sed -E 's|^.+@(.+)$|\1|')"
+
+  local cmd_path
+  cmd_path=$(command -v "${tool_name}")
+  if [[ $? -eq 0 ]] ; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)